common-auth/src/main/java/com/hortonworks/registries/auth/server/JWTAuthenticationHandler.java

```diff
@@ -368,4 +387,33 @@ protected boolean validateExpiration(SignedJWT jwtToken) {
         }
         return valid;
     }
+
+     /**
+     * This method parses the User-Agent String and returns whether or not it
+     * refers to a browser.
+     * <p>
+     * A User-Agent String is considered to be a browser if it does not contain
+     * any of the values from non-browser.user-agents; the default
+     * behavior is to consider everything a browser unless it contains one of:
+     * "java", "curl", "wget", "perl" or "Jersey".  Subclasses can optionally override
+     * this method to use different behavior.
+     *
+     * @param userAgent The User-Agent String, or null if there isn't one
+     * @return true if the User-Agent String refers to a browser, false if not
+     */
+    private boolean isBrowser(String userAgent) {
```

Comment:
Can we avoid duplicating this method between handler classes? maybe we can move to util class?